packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -39,32 +39,30 @@ class RawKeyEventDataLinux extends RawKeyEventData {
   /// (GLFW, GTK, QT, etc) may have a different key code mapping.
   final KeyHelper keyHelper;
 
+  /// The name of the printable key. This is typically the character that [keyCode] would
```

Comment:
This isn't really a name. It's the set of Unicode scalar values generated by a single keystroke.

Reply:
Done